src/v3/src/components/Icon/OdysseyIcons.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+export {
+  AppsIcon,
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  CheckCircleFilledIcon,
+  CloseIcon,
+  DevicesIcon,
+  GlobeIcon,
+  HideIcon,
+  InformationCircleIcon,
+  ShowIcon,
+  UserIcon,
+} from '@okta/odyssey-react-mui/icons';
```

Comment:
Is there any reason we don't directly import from `@okta/odyssey-react-mui/icons` in each place we need an icon?

Reply:
No reason. Changed to direct imports